src/CachePlugin.php

```diff
@@ -58,6 +58,19 @@ public function __construct(CacheItemPoolInterface $pool, StreamFactory $streamF
         $this->config = $optionsResolver->resolve($config);
     }
 
+    public static function clientCache(CacheItemPoolInterface $pool, StreamFactory $streamFactory, array $config = [])
+    {
+        // Allow caching of private requests
+        $config['excluded_directives'] = ['no-cache'];
```

Comment:
I'm also more in favor of separate options. Because I would avoid mixing option types bool/array